blockchain/chain_sync/Cargo.toml

```diff
@@ -52,7 +52,7 @@ time = { version = "0.3", features = ["serde"] }
 prometheus = { version = "0.12.0", features = ["process"] }
 lazy_static = "1.4.0"
 statediff = { path = "../../utils/statediff", optional = true }
-fvm = "1.0"
+fvm = "1.1"
```

Comment:
If you change the `fvm` version then you have to change it everywhere. You missed it in `blockchain/state_manager/Cargo.toml`. I don't think the version bump is necessary for this PR so perhaps we can leave it at `1.0`.

Reply:
apart from [2.] (as not all crates are in the registry), I hope everything is solved